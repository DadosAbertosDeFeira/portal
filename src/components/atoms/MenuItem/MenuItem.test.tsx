import { render } from "@testing-library/react";

import { MenuItem, type MenuItemProps } from ".";

describe("MenuItem", () => {
  const makeSut = ({ ...props }: Partial<MenuItemProps>) => {
    return render(<MenuItem {...props} />);
  };

  it("renders correctly", () => {
    const { asFragment } = makeSut({});

    expect(asFragment).toMatchSnapshot();
  });
});
