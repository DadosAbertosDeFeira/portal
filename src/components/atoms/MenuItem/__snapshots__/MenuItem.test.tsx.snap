// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuItem renders correctly 1`] = `[Function]`;

exports[`MenuItem should render focused correctly 1`] = `[Function]`;
